### PR TITLE
add Tests for Adress Packge

### PR DESCRIPTION
**Pull request by josephbate**
# Pull Requests Requirements

- [ ] The PR title includes a brief description of the work done, including the
      Issue number if applicable.
- [ ] The PR includes a video showing the changes for the work done.
- [ ] The PR title follows conventional commit label standards.
- [ ] The changes confirm to the OpenElis Global x3 Styleguide and design
      documentation.
- [ ] The changes include tests or are validated by existing tests.
- [ ] I have read and agree to the Contributing Guidelines of this project.

## Summary
Added more tests for address package
## Screenshots

[Add relevant screenshots here if applicable]

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## Other

[Add any additional information or notes here]

### CODE REVIEW

File: src/test/java/org/openelisglobal/address/AddressPartServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    public void updateAddressPart_shouldUpdateAddressPart() throws Exception {
+        AddressPart part = new AddressPart();
+        part.setPartName("PartName");
+        part.setDisplayOrder("022");
+
+        Assert.assertEquals(0, partService.getAll().size());
+
+        String partId = partService.insert(part);
+        AddressPart savedPart = partService.get(partId);
+        savedPart.setPartName("upadtedName");
```
Comment (mherman22) [inline src/test/java/org/openelisglobal/address/AddressPartServiceTest.java]:
```suggestion
        savedPart.setPartName("updatedName");
```

File: src/test/java/org/openelisglobal/address/AddressPartServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        partService.save(savedPart);
+
+        Assert.assertEquals("upadtedName", savedPart.getPartName());
```
Comment (mherman22) [inline src/test/java/org/openelisglobal/address/AddressPartServiceTest.java]:
```suggestion
        Assert.assertEquals("updatedName", savedPart.getPartName());
```

File: src/test/java/org/openelisglobal/address/AddressPartServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    public void deleteAddressPart_shouldDeleteAddressPart() throws Exception {
+        AddressPart part = new AddressPart();
+        part.setPartName("PartName");
+        part.setDisplayOrder("022");
+
+        Assert.assertEquals(0, partService.getAll().size());
+
+        String partId = partService.insert(part);
+        AddressPart savedPart = partService.get(partId);
+        savedPart.setPartName("upadtedName");
```
Comment (mherman22) [inline src/test/java/org/openelisglobal/address/AddressPartServiceTest.java]:
```suggestion
        savedPart.setPartName("updatedName");
```

Comment (mherman22) [reply src/test/java/org/openelisglobal/address/AddressPartServiceTest.java]: Why are you setting a new name here? shouldn't you just delete?

File: src/test/java/org/openelisglobal/address/PersonAdressServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+        PersonAddress personAddress = new PersonAddress();
+        personAddress.setAddressPartId(partId);
+        personAddress.setPersonId(personId);
+        personAddress.setType("B");
+        personAddress.setValue("123");
+
+        Assert.assertEquals(0, pAddressService.getAll().size());
+
+        AddressPK savedPAID = pAddressService.insert(personAddress);
```
Comment (mherman22) [inline src/test/java/org/openelisglobal/address/PersonAdressServiceTest.java]:
```suggestion
        AddressPK savedAddressPK = pAddressService.insert(personAddress);
```

**Review by mherman22 — CHANGES_REQUESTED** [review]

**Comment by mherman22** [comment]
can you close https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/1312 if you nolonger need it?

**Comment by mozzy11** [comment]
Thanks @mherman22 .
@josephbate can you look at the above comments ??

**Review by mherman22 — APPROVED** [review]
LGTM, thanks @josephbate

**Comment by josephbate** [comment]
> LGTM, thanks @josephbate

thank you too

**Pull request closed without merging**